### ISSUE TITLE
Expose original React component in RelayContainer

### ISSUE THREAD

**Issue by vladar** [facebook/relay#154]
Actually I need a way to access static methods of original component and looks like there is no way to access them when Component is wrapped with RelayContainer?

**Comment by vladar** [comment]
I realized that it is possible to assign static methods to Container instead of Component:

```javascript
let PageContainer = Relay.createContainer(...);
PageContainer.staticMethod = () => {}
```

Not that pretty, but works.

**Comment by cpojer** [comment]
Doing all this kind of forwarding makes static analysis harder. Originally we used mixins instead of containers and statics were automatically available. When we moved to containers we used to forward statics (and instance methods) because that made it easier to move to the new implementation. We realized that doing the forwarding for static fields wasn't very useful, so we moved them to the containers and removed support for the feature. This is the recommended solution.

We are doing the same for instance methods internally. There is a forked file called RelayContainerProxy and we are working on getting rid of it internally – we are hoping that React will provide a solution for this in the future.
